libcaliptra/inc/caliptra_types.h

```diff
@@ -365,6 +373,8 @@ struct caliptra_invoke_dpe_req {
     struct caliptra_req_header hdr;
     uint32_t data_size;
     union {
+        struct dpe_cmd_hdr                      cmd_hdr;
```

Comment:
Why is it required in here?

Reply:
Second thought, I see usefulness of this field. Ignore this comment please.

Reply:
Okay cool, I figured it would be easier for working with a command generically